backend/app/api/v1/endpoints/chat.py

```
@@ -1,49 +1,46 @@
-            message=str(ve),
-            code=400
+            message="Chat response generated successfully",
         )
-    except Exception as e:
-        return ResponseHandler.error_response(
-            message=str(e),
-            code=500
-        )
\ No newline at end of file
+    except BadRequestException as ve:
+        raise ve
```

Ruff (0.14.10)

Remove exception handler; error is immediately re-raised

(TRY203)